Bug fixes should be included on main and release branch (#657)
kyma-project/community: docs/guidelines/releases-guidelines/02-kyma-release-process.md

   > **NOTE:** Every developer who is introducing changes to the specific version can perform steps 1-4.

1. Create a feature-branch based on the given `release-{RELEASE}` branch you want to extend. Add your changes and create a Pull Request.

>    Usually fix is made first on `main` branch and then backported into `release-{RELEASE}` branch. Please make sure that changes are included on `main` branch.

Reviewer: ```suggestion
   Usually, a fix is done first on the `main` branch and then backported into the `release-{RELEASE}` branch. Make sure that the changes are included in the `main` branch.
```
